### PR TITLE
Update to use kubernetes credential store

### PR DESCRIPTION
**Pull request by paulczar**
By using kuberentes secrets we only need to update the `secrets.yaml` rather
than each kubernetes manifest file.

### CODE REVIEW

**Comment by AnthonyAmanse** [comment]
LGTM

**Comment by Tomcli** [comment]
LGTM

File: reviews-new.yaml [hunk 1]
```diff
@@ -56,14 +68,26 @@ spec:
...
+          - name: MYSQL_DB_USER
+            valueFrom:
+              secretKeyRef:
+                name: demo-credentials
+                key: username
+          - name: MYSQL_DB_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: demo-credentials
+                key: username
```
Comment (AnthonyAmanse) [inline reviews-new.yaml]: this should be `key: password`

File: reviews-new.yaml [hunk 2]
```diff
@@ -28,14 +28,26 @@ spec:
...
```
Comment (AnthonyAmanse) [inline reviews-new.yaml]: this should be `key: password`

Comment (paulczar) [inline reviews-new.yaml]: hahaha good catch FML

**Pull request merged**